docs/getting-started/benchmarking.md

```diff
@@ -0,0 +1,96 @@
+---
```

Comment:
I think that it would be nice to have figures for some if not all tasks to make it easier to understand.

Reply:
I know. It's in my to-do. I have figures for all of them, but wanted to get this merged before the weekend

Reply:
I see. Then we can leave it for another PR.